mobile/pubspec.yaml

```diff
@@ -1,5 +1,7 @@
 name: app_rating
-description: A new Flutter package project.
+description: >
+  Gather feedback with a 5-star rating or Yes/No survey. Includes device analytics and app store rating prompts for positive reviews.
+repository: https://github.com/leancodepl/app-rating/tree/main/mobile
```

Comment:
```suggestion
repository: https://github.com/leancodepl/app-rating
```

Reply:
If the actual directory is other, pass the specific path (like what you had here initially) to `homepage`. `repository` must be repo root